UI/config.csv

```diff
@@ -1167,10 +1167,10 @@ Config_085_Hint#0,"""Set whether to show the interface <br>(including windows) i
 Config_085_Choices_0_Switch#0,"""Display Windows"""
 Config_085_Choices_1_Switch#0,"""Hide Windows"""
 Config_085_Hint_Switch#0,"""Set whether to show the interface <br>(including windows) in screenshots.<br>※Only applies to in-game functions."""
-Config_086_Name#0,"""Synchronized Physique Editing Settings"""
-Config_086_Choices_0#0,"""Synchronized Physique Editing"""
-Config_086_Choices_1#0,"""Separate Physique Editing"""
-Config_086_Hint#0,"""Set whether you want your Casts' human and robotic bodies to share the same proportions, or whether you want to edit them separately.<br><br>Humanoid-type and Cast bodies are synchronized by default. However, you can individually edit them by selecting \""Separate Physique Editing\""."""
+Config_086_Name#0,"""Body Shape Synchronization"""
+Config_086_Choices_0#0,"""Synchronized Body Shapes"""
+Config_086_Choices_1#0,"""Edit Body Shapes Individually"""
+Config_086_Hint#0,"""The Body Shapes of Body Types across race/gender<br>are synchronized by default. You can edit them<br>individually with \""Edit Body Shapes Individually\"".<br><br>Body Types will change depending on the worn<br>costume but Body Shape proportions will not.<br>With this setting, you can set Body Shapes<br>individually for each Body Type."""
```

Comment:
This menu doesn't need manual line breaks unless you're splitting sentences to make a paragraph.